test/Microsoft.DocAsCode.Build.Engine.Tests/IncrementalBuildTest.cs

```diff
@@ -1974,7 +1974,7 @@ public void TestCaseNotMatchIncludeFileWithInvalidBookmarkReplayLog()
                         templateFolder: templateFolder,
                         intermediateFolder: intermediateFolder);
                     Assert.Equal(2, Listener.Items.Count);
-                    Assert.NotNull(Listener.Items.FirstOrDefault(s => s.Message.StartsWith("Illegal link: `[link](#invalid)` -- missing bookmark")));
```

Comment:
do we want to remove this?

Reply:
@herohua It's better to check if the log item is what we want or not.

Reply:
sorry I missed that. thanks for letting me know!